tests/unit/test_neptune.py

```diff
@@ -130,21 +130,21 @@ def test_gremlin_query_vertices(neptune_endpoint, neptune_port) -> Dict[str, Any
 
     wr.neptune.execute_gremlin(client, f"g.addV().property(T.id, '{uuid.uuid4()}')")
     df = wr.neptune.execute_gremlin(client, "g.V().limit(1)")
-    assert df.shape == (1, 2)
+    assert df.shape == (1, 3)
```

Comment:
Is this because of the empty property key? We did not upgrade gremlin so I'm curious does it behave different in Python 3.12 or what is the reason?